src/rgw/rgw_rest_conn.h

```diff
@@ -90,6 +90,8 @@ class RGWRESTConn
   }
   size_t get_endpoint_count() const { return endpoints.size(); }
 
+  virtual void populate_params(param_vec_t& params, const rgw_user *uid, const string& zonegroup);
```

Comment:
@Leeshine `populate_params()` is too generic, let's in addition have `populate_zonegroup()` and use that one instead.